packages/cli/tests/e2e/m365/DebugM365Tab.tests.ts

```diff
@@ -34,65 +30,72 @@ describe("Debug V3 m365-tab template", () => {
     if (!isV3Enabled()) {
       this.skip();
     }
-
-    // clean up
-    const context = await readContextMultiEnvV3(projectPath, "local");
-    if (context?.TEAMS_APP_ID) {
-      await deleteTeamsApp(context.TEAMS_APP_ID);
-    }
-    if (context?.AAD_APP_OBJECT_ID) {
-      await deleteAadAppByObjectId(context.AAD_APP_OBJECT_ID);
-    }
-    await cleanUpLocalProject(projectPath);
+    await Cleaner.clean(projectPath);
   });
 
-  it("happy path: provision and deploy", { testPlanCaseId: 17449535 }, async function () {
+  it("- happy path: provision and deploy", { testPlanCaseId: 17449535 }, async function () {
```

Comment:
Just curious why add "- "?

Reply:
Just for showing test report more beautiful because the case filed in the report will combine the suite name and it name together.
![image](https://user-images.githubusercontent.com/15262146/228193819-2c4b89e7-2110-4839-92ae-3b1a3eadf55e.png)

Reply:
Is it possible to do that in pipeline instead of in code?

Reply:
It's hard to normalize the name in pipeline. I think we may normalize the name by other way. That is unnecessary and change it back.